main-project/src/boards/entity/board.bookmark.entity.ts

```diff
@@ -0,0 +1,8 @@
+import { BaseEntity, Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+// fk없음, entity취합 후 생성예정
+@Entity()
+export class Board_bookmark extends BaseEntity {
```

Comment:
```suggestion
@Entity('board_bookmarks')
export class BoardBookmark extends BaseEntity {
```
이런 네이밍을 사용해보는 건 어떤가요?
클래스 이름에 언더바가 들어가는 것은 좋지 않습니다.